Resfeber/App Lifecycle/SceneDelegate.swift

```diff
@@ -13,6 +13,24 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
     
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        
+        guard let scene = (scene as? UIWindowScene) else { return }
+        
+        window = UIWindow(windowScene: scene)
+        
+        let tabBar = MainTabBarController()
+        let exploreVC = ExploreViewController()
+        let favoritesVC = FavoritesViewController()
+        let itineraryVC = ItineraryViewController()
+        
+        tabBar.setViewControllers([exploreVC, favoritesVC, itineraryVC], animated: false)
```

Comment:
Looks good. In the future we might want to move some of this tab bar setup code out of the SceneDelegate and into the MainTabBarController.